account-setup.rst

```diff
@@ -61,6 +61,7 @@ Add a secondary user
 We recommend that all customers with commercial subscriptions have a secondary user on the Ubuntu Pro dashboard, to ensure continuity of access.
 
 Check *Account users* from the main menu. If a secondary user is not already listed, add one now.
+..  image:: /images/pro-adduser.png
```

Comment:
```suggestion
Check *Account users* from the main menu. If a secondary user is not already listed, add one now.

.. image:: images/pro-adduser.png
```

Reply:
I think this is the part that's causing the build to fail. Usually, restructuredtext directives need a blank/empty line before and after, otherwise it's treated as part of the previous paragraph (the spell checker was picking up "adduser" and "png" so it's not recognising these as part of the file path).

If this doesn't work let me know!